tools/clang/unittests/HLSL/ShaderOpTest.cpp

```diff
@@ -902,6 +902,10 @@ void ShaderOpTest::RunCommandList() {
 
 #if defined(NTDDI_WIN10_VB) && WDK_NTDDI_VERSION >= NTDDI_WIN10_VB
     if (m_pShaderOp->MS) {
+#ifndef NDEBUG
+      D3D12_FEATURE_DATA_D3D12_OPTIONS7 O7;
+      DXASSERT_LOCALVAR(O7, SUCCEEDED(m_pDevice->CheckFeatureSupport((D3D12_FEATURE)D3D12_FEATURE_D3D12_OPTIONS7, &O7, sizeof(O7))), "mesh shader test enabled on platform without mesh support");
+#endif
       ID3D12GraphicsCommandList6 *pList6 = m_CommandList.List.p;
```

Comment:
Does this actually compile? Shouldn't this need to be a `QueryInterface` instead of an unsafe cast?

Reply:
Woops. Missed a line from a manual copy